docs/reference/contributing/target/usb.md

```diff
@@ -1,25 +1,50 @@
-### USB
+### USB device
 
-[Include a brief description here.]
+The HAL USBPhy API provides physical access to the USB bus in the role of a USB device. Implementing this API enables the use of any supported class driver, such as CDC, HID and MSD.
 
 #### Assumptions
 
 ##### Defined behavior
 
-[Include any defined behavior in bullet format here.]
+- You can use any endpoint configurations that fits in the parameters of the table returned by `USBPhy::endpoint_table`.
+- You can use all endpoints in any valid endpoint configuration concurrently.
+- The device supports use of at least one control, bulk, interrupt and isochronous in each direction at the same time - at least 8 endpoints.
+- The device supports all standard endpoint sizes (wMaxPacketSize).
+- The device can handle an interrupt latency of at least 100ms if reset is not being performed and address is not being set.
```

Comment:
Query: Who or what is not performing the reset? Also, who or what is not sending the address? You, the user?